skaffold.yaml

```diff
@@ -1165,17 +1154,7 @@ profiles:
     value:
     - bash
     - -ec
-    - |
-      echo "Wait until seed is ready"
-      for i in `seq 1 30`;
-      do
-        if kubectl --kubeconfig=$GARDENER_LOCAL_KUBECONFIG get seed local2 2> /dev/null; then
-          break
-        fi
-        echo "Wait until seed gets created by gardenlet"
-        sleep 2
-      done
-      kubectl --kubeconfig=$GARDENER_LOCAL_KUBECONFIG wait --for=condition=gardenletready --for=condition=extensionsready --for=condition=bootstrapped seed local2 --timeout=5m
+    - hack/usage/wait.sh seed local2 GardenletReady Bootstrapped SeedSystemComponentsHealthy ExtensionsReady BackupBucketsReady
```

Comment:
The `hack/usage/wait.sh` script probably needs a `KUBECONFIG` parameter. The script needs to query the `garden cluster` but in this case the kind2 cluster is not the garden cluster, but a standalone seed.
This makes the migration tests failing.

Reply:
This might work too
``` suggestion
    - KUBECONFIG=$GARDENER_LOCAL_KUBECONFIG hack/usage/wait.sh seed local2 GardenletReady Bootstrapped SeedSystemComponentsHealthy ExtensionsReady BackupBucketsReady
```